libs/perun/openapi/src/lib/api/databaseManager.service.ts

```diff
@@ -448,7 +445,6 @@ export class DatabaseManagerService {
       path[1] = 'non';
       helperUrl.pathname = path.join('/');
       requestUrl = helperUrl.toString();
-      console.log(requestUrl);
```

Comment:
Is there any specific reason why there were all those console logs before and were removed?

Reply:
The PR in which 'useNon' was implemented was probably merged with openapi generated by the test template. This isn't a big issue, since a new generation of openapi with the correct template (which was merged) will no longer have this log (hence these changes).